TaskER_FlowChart.md

```
@@ -682,7 +682,164 @@ Can be entry point or follow any block
+|-----------|------|----------|-------------|
+| `task` | Integer | ✅ Yes | Unique task identifier |
+| `type` | String | ✅ Yes | Must be "parallel" |
+| `hostnames` | String | ✅ Yes | Comma-separated hostnames (2-1000) |
+| `command` | String | ✅ Yes | Command to execute on all hosts |
+| `arguments` | String | ❌ Optional | Command arguments |
+| `exec` | String | ❌ Optional | Execution method (local/pbrun/p7s) |
+| `max_parallel` | Integer | ❌ Optional | Max concurrent hosts (1-50, default: 8) |
+| `retry_count` | Integer | ❌ Optional | Number of retry attempts (1-1000, default: 1) |
+| `retry_delay` | Integer | ❌ Optional | Delay between retries (0-300 seconds, default: 1) |
+
+### Example
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)